.github/workflows/pr_crictl.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         fi;
         echo "rune_test=$rune_test" >> $GITHUB_ENV
 
+    # FIXME: use make package command to build shelter, inclavared and enclave-tls packages
```

Comment:
Change to "FIXME: Don't run `make package` to build  shelter, inclavared and enclave-tls due to bug"